CMakeLists.txt

```diff
@@ -74,7 +76,7 @@ if(WIN32)
     )
 endif()
 target_link_libraries(${PROJECT_NAME} PRIVATE
-    Qt${QT_VERSION_MAJOR}::GuiPrivate
+    Qt${QT_VERSION_MAJOR}::GuiPrivate Qt${QT_VERSION_MAJOR}::Quick
```

Comment:
Same. Don't link to it unconditionally.

Reply:
Fixed in c287322

Reply:
I'm afraid not.

Reply:
Seems not fixed.

Reply:
What about this one? Seems c287322 not fixed this.